src/renderer/drawers/Drawer.js

```diff
@@ -60,31 +67,52 @@ export const Drawer = () => {
 
   const onRequestClose = useCallback(() => setDrawer(), [setDrawer]);
 
+  const sideDrawerRef = useRef();
+  const onResize = useCallback(e => {
+    // NB Passing this down to the Component as a prop allows us to have sticky elements
+    // by setting a fixed height instead of a percentage of the container, if you are reading
+    // this and have a better solution, treat yourself. -77 due to drawer wrapper padding.s
+    if (!sideDrawerRef.current) return;
+    setHeight(sideDrawerRef.current.clientHeight - 77);
```

Comment:
why not use css for this :thinking: are you loading an iframe inside the drawer or a simple `calc(100vh - 77)` can't work in this case ? :thinking:

Reply:
couldn't make it work with the sticky elements, it was @IAmMorrow who took over afterwards but if you can make it work without this hack, feel free

Reply:
Tried to understand why we needed that, couldn't notice a difference with or without all of the additional stuff in the Drawer even when resizing. 
Removed it for now, I'll just cherry pick the commit and fix it properly with the `calc` if I can find what the problem was at some point 🤷‍♂️